src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -293,9 +293,13 @@ def load_and_exec_node(
                 node_name=node_name,
                 flow_file=flow_file,
             )
-
-        inputs_with_default_value = FlowExecutor._apply_default_value_for_input(flow.inputs, flow_inputs)
-        inputs = load_multimedia_data(flow.inputs, inputs_with_default_value)
+        node_reference_flow_inputs_keys = []
+        for _, value in node.inputs.items():
+            if value.value_type == InputValueType.FLOW_INPUT:
+                node_reference_flow_inputs_keys.append(value.value)
+        inputs_with_default_value = FlowExecutor._apply_default_value_for_input(
+            flow.inputs, flow_inputs, node_reference_flow_inputs_keys)
+        inputs = load_multimedia_data(flow.inputs, inputs_with_default_value, node_reference_flow_inputs_keys)
         dependency_nodes_outputs = load_multimedia_data_recursively(dependency_nodes_outputs)
         converted_flow_inputs_for_node = FlowValidator.convert_flow_inputs_for_node(flow, node, inputs)
```

Comment:
Let's move `inputs = load_multimedia_data(flow.inputs, inputs_with_default_value, node_reference_flow_inputs_keys)`
behind `converted_flow_inputs_for_node = FlowValidator.convert_flow_inputs_for_node(flow, node, inputs)`
There is validation logic about flow input in `FlowValidator`

Reply:
fixed